Scripts/extract_release_notes.rb

```diff
@@ -0,0 +1,81 @@
+# Parses the release notes file to extract the current version information and
+# puts it to STDOUT.
+#
+# To update the release notes for localization:
+#
+# ruby ./this_script >| Simplenote/Resources/release_notes.txt
+#
+# To generate the App Store Connect release message:
+#
+# ruby ./this_script | pbcopy
+#
+# To generate the GitHub and App Center release message:
+#
+# ruby ./this_script -k | pbcopy
+
+GITHUB_URL = 'https://github.com/Automattic/simplenote-ios'
+
+RELEASE_NOTES_FILE = 'RELEASE-NOTES.txt'
+NOTES = File.read(RELEASE_NOTES_FILE)
+lines = NOTES.lines
+
+def replace_pr_number_with_markdown_link(string)
+  string.gsub(/\#\d*$/) do |pr_number|
+    "[#{GITHUB_URL}/pull/#{pr_number.gsub('#', '')}](#{pr_number})"
+  end
+end
+
+# This is a very bare bone option parsing. It does the job for this simple use
+# case, but it should not be built upon.
+#
+# If you plan to add more options, please consider using a gem to manage them
+# properly.
```

Comment:
note: actually `OptionParser` is part of ruby's stdlib, so no need to use an external gem if you wanted to do it properly 😛 – https://ruby-doc.org/stdlib-2.7.2/libdoc/optparse/rdoc/OptionParser.html

Reply:
🎉